src/test/java/com/example/universe/simulator/entityservice/integration/GalaxyIntegrationTest.java

```diff
@@ -63,14 +64,21 @@ void test() throws Exception {
         resultList = objectMapper.readValue(response.getContentAsString(), new TypeReference<>() {});
         assertThat(resultList.getContent()).hasSize(2);
 
-        // -----------------------------------should return entity-----------------------------------
+        // -----------------------------------should return entity and store it in cache-----------------------------------
 
         // when
         response = performRequest(get("/galaxy/get/{id}", addedDto1.getId()));
         // then
         GalaxyDto resultDto = objectMapper.readValue(response.getContentAsString(), GalaxyDto.class);
         assertThat(resultDto).isEqualTo(addedDto1);
 
+        //check if resultDto got into cache
```

Comment:
remove unnecessary comment

Reply:
Okay, But why is it unnecessary?

Reply:
You can say that on any comment written in this class. Could you be more specific?

Reply:
ok, here you are commenting your code, but your code says the same, why to comment then?

Reply:
as to the rest of comments, if this miles long method will be split in several small methods with self explanatory names, then all comments could be easily removed